### PR TITLE
POPS-1111 Remove Env Dependency

### PR DESCRIPTION
**Pull request by Hbbb**
* Removes dependency on env vars in CLI
* Refactors tuber config into `config` package

### CODE REVIEW

**Review by quinn-freshly — APPROVED** [review]

File: pkg/config/main.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package config
```
Comment (jkminneti) [inline pkg/config/main.go:1]: can we rename this file `config.go` for my sanity?

File: cmd/config.go [hunk 2]
```diff
@@ -14,7 +15,7 @@ var configCmd = &cobra.Command{
 	Use:          "config",
 	Short:        "open local tuber config in your default editor",
 	Args:         cobra.NoArgs,
-	RunE:         config,
+	RunE:         writeConfig,
```
Comment (jkminneti) [inline cmd/config.go:18]: it's more like `openConfig` but obviously pedantic c6 or whatever

**Review by quinn-freshly — CHANGES_REQUESTED** [review]
seems to have build errors

File: pkg/config/main.go (hunk 1 above)
Comment (Hbbb) [reply pkg/config/main.go:1]: Absolutely.

**Pull request merged**